apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -95,6 +95,7 @@ module.exports = class Filepanel extends ViewPlugin {
   }
 
   getAvailableWorkspaceName (name) {
+    if(!this.workspaces) return null
```

Comment:
pehaps this should just return `name + ' - 1 '`. because i m not sure how the caller of `getAvailableWorkspaceName` will react if it returns null.
or handle the case where it returns null.